students.md

```diff
@@ -2,45 +2,45 @@
 
 Number | Name      | Github | Tasks done | Summary
 -------|-----------|---------|---------|---------
-1 | Anna Boyko| [git](https://github.com/Boyko-Anna) | Task 8 | self learned HTML, CSS, C# courses Cyber Bionics, some Ruby courses, JS self learning, interested in remote work.
-2 | Daria Shvydka | [git](https://github.com/DariaShvydka) | Task 9  | HTML, CSS, SASS production experience ~ 1 year. JS - self learning.
-3 | Igor Kurkov | [git](https://github.com/IgorKurkov) | Task 7 | Created https://igorkurkov.github.io/kottans-stats/, self learning php, js, Drupal
+1 | Anna Boyko| [git](https://github.com/Boyko-Anna) | Task 8 | self-learned HTML, CSS, C# courses Cyber Bionics, some Ruby courses, JS self-learning, interested in remote work.
+2 | Daria Shvydka | [git](https://github.com/DariaShvydka) | Task 9  | HTML, CSS, SASS production experience ~ 1 year. JS - self-learning.
+3 | Igor Kurkov | [git](https://github.com/IgorKurkov) | Task 7 | Created https://igorkurkov.github.io/kottans-stats/, self-learning php, js, Drupal
 4 | Maxim | [git](https://github.com/cidre) | Task 8 | 3rd time trying to get on the course, no prod experience
 5 | Oleksii Nelin | [git](https://github.com/Xsorter) | Task 10  | ~1 year of production experience with JS, JQuery, CSS, HTML
 6 | Miroslav | [git](https://github.com/iammiro/kottans_frontend) | Task 10  | Some prod experience, layout sites, JQuery
 7 | Anna Stepashko | [git](https://github.com/xandzia) | Task 8 | CSS, HTML, JQuery self education, no production experience, interested in start working
 8 | Yevhen Orlov | [git](https://github.com/yevhenorlov) | Task 10 | 1,5 years production experience HTML, SASS, jQuery, want more serious Front End
-9 | Iaroslav Garmash | [git](https://github.com/feroxes) | Task 7 | No production experience, self learned HTML, CSS
-10 | Olexiy Letushev | [git](https://github.com/Letushev) | Task 9 | No production experience, student of Shevchenko University, HTML, CSS self learned and JS started with this course
+9 | Iaroslav Garmash | [git](https://github.com/feroxes) | Task 7 | No production experience, self-learned HTML, CSS
+10 | Olexiy Letushev | [git](https://github.com/Letushev) | Task 9 | No production experience, student of Shevchenko University, HTML, CSS self-learned and JS started with this course
 11 | Mykhailo Hanol | [git](https://github.com/ganolmc) | Task 8 | Some freelance experience about 5 years ago, Bionics University intern, interested in learning JS more
 12 | Igor Surzhyk | [git](https://github.com/isurzhyk) | Task 7 | No experience with JS, HTML, CSS, always wanted to create products
-13 | Tania Maslivets | [git](https://github.com/Masmik) | Task 9 | GOIT, self learning, second try
+13 | Tania Maslivets | [git](https://github.com/Masmik) | Task 9 | GOIT, self-earning, second try
 14 | Viktoriia Buts | [git](https://github.com/viktoriiab) | Task 8 | Student in KPI, want to work in frontend, 3-rd try
```

Comment:
learning => earning. typo?

Reply:
Oh, I see. I will fix

Reply:
you can push one more commit to your branch and it will be added to PR automatically (just in case)

Reply:
Ok, I will test now. Don't merge pls. Thank you!

Reply:
I can't push to the branch
http://take.ms/GUxlM

Reply:
Looks like I have fixed. 
@AMashoshyna can you check and merge if everything is okey?